markup/tableofcontents/tableofcontents.go

```diff
@@ -40,19 +40,19 @@ type Root struct {
 }
 
 // AddAt adds the header into the given location.
-func (toc *Root) AddAt(h Header, y, x int) {
-	for i := len(toc.Headers); i <= y; i++ {
+func (toc *Root) AddAt(h Header, row, level int) {
```

Comment:
This does not look related.

Reply:
It's only related in the sense that it's called in
https://github.com/gohugoio/hugo/blob/64fcb35c13829bdc50d0f92628cc60674ed5a193/markup/asciidocext/convert.go#L276-L279

And I had to read it to understand how the ToC works in Hugo. Parameters named `y` and `x` didn't help with that.

I'm happy to revert this change in this PR if you prefer, but I think you should consider adding it in a future commit as it helps with the readability and it's consistent with the variable names used in the only other place it's called:
https://github.com/gohugoio/hugo/blob/64fcb35c13829bdc50d0f92628cc60674ed5a193/markup/goldmark/toc.go#L58